docs/Compile/BuildWindows.md

```diff
@@ -91,8 +92,8 @@ If it is not in the same level directory, you must specify the CMake parameters:
 
 #### libvncserver
 - Compile from sorce code
-  + Source code location: [https://github.com/LibVNC/libvncserver](https://github.com/LibVNC/libvncserver)  
-  It is recommended to use patches: https://github.com/KangLin/libvncserver
+  + Source-code location: [https://github.com/LibVNC/libvncserver](https://github.com/LibVNC/libvncserver)  
+  It is recommended to use the patches from: https://github.com/KangLin/libvncserver
   + Specify CMake parameters: -Dvncclient_DIR=[libvncserver installation path]/lib/cmake/LibVncServer
```

Comment:
Not sure "lib/cmake/LibVncServer" is correct

Reply:
I am will chanage to libVNCServer.